### ISSUE TITLE
The given path's format is not supported

### ISSUE THREAD

**Issue by AaronChrisCyber** [mandiant/flare-vm#449]
I'm not sure why I'm unable to download Flare. I've tried multiple times and went through the logs as well and I'm not getting any specific reason as to why It won't come up. 

![Screenshot 2023-04-30 155452](https://user-images.githubusercontent.com/132229734/235374967-d1e013df-91c0-492c-81c0-728e34461d5e.png)

**Comment by mr-tz** [comment]
Can you share those logs and the system info (see the bug issue template)?

**Comment by Ana06** [comment]
Can you please add details about how you are trying to install flare-vm? It seems it is not looking for the package in the chocolatey feed.

**Comment by AaronChrisCyber** [comment]
Hi Ana,

So these are the steps I've taken as directed by the PMAT course I'm
taking. Once it downloads the tools aren't available in the tools folder
but the txt file lists all of the available packages.


   - Download Windows Terminal:
      - Download the VCLibs package. In a PowerShell window, run: wget
      https://aka.ms/Microsoft.VCLibs.x64.14.00.Desktop.appx
      -usebasicparsing -o VCLibs.appx
      - Download the Windows Terminal MSIX bundle from the provided link: wget

      https://github.com/microsoft/terminal/releases/download/v1.15.3465.0/Microsoft.WindowsTerminal_Win10_1.15.3465.0_8wekyb3d8bbwe.msixbundle
      -UseBasicParsing -o winterminal.msixbundle
      - In a PowerShell admin window, add the VCLibs package: Add-AppxPackage
      [C:\path\to\downloaded\VCLibs.appx]
      - In a PowerShell admin window, run: Add-AppxPackage
      [C:\path\to\downloaded\winterminal.msixbundle]
      - (Optional) Pin Windows Terminal to the task bar
   - Disable proxy auto detect setting:
      - In the Windows search bar, search “proxy settings”,
      - Switch "Automatically detect settings" button off
   - Disable Tamper Protection
      - Search "Defender", open Defender settings and set all Defender
      Settings to off
   - Disable AV/Defender in GPO
      - In Windows search bar, search "group policy"
      - In GPO, navigate to → Administrative Templates → Windows Components
      → Microsoft Defender Antivirus → Enable “Turn off Microsoft Defender
      Antivirus”
   - Disable Windows Firewall
      - GPO → Administrative Templates → Network → Network Connections →
      Windows Defender Firewall → Domain Profile → Disable “Protect All Network
      Connections”
      - Do the same but for the Standard profile
   - TAKE A SNAPSHOT!
   - Download and install FLARE-VM:
      - In PowerShell Admin prompt, run: (New-Object
      net.webclient).DownloadFile('
      https://raw.githubusercontent.com/mandiant/flare-vm/main/install.ps1
      ',"$([Environment]::GetFolderPath("Desktop"))\install.ps1")
      - Change directories to the Desktop
      - Run: Unblock-File .\install.ps1
      - Run: Set-ExecutionPolicy Unrestricted
      - Accept the prompt to set the ExecPol to unrestricted if one appears
      - Run: .\install.ps1 -customConfig
      https://raw.githubusercontent.com/HuskyHacks/PMAT-labs/main/config.xml
      - Follow the rest of the prompts and continue with the installation

**Comment by Ana06** [comment]
Can you please retry with the last version of flare-vm? if it still fails, can you please also share the installation log/s and the system info (see the bug issue template)?

**Comment by Ana06** [comment]
Closing due to inactivity. Please open a new issue if you think this still occurs. When reporting bugs use the issue template and provide all the required information including the output of `VM-Get-Host-Info` .
